src/main/java/net/minestom/server/entity/metadata/animal/GoatMeta.java

```diff
@@ -19,4 +19,20 @@ public boolean isScreaming() {
     public void setScreaming(boolean screaming) {
         metadata.setIndex(OFFSET, Metadata.Boolean(screaming));
     }
+
+    public boolean hasLeftHorn() {
```

Comment:
need to update MAX_OFFSET for these new fields

Reply:
Sorry for letting you wait. GitHub really needs a reminders feature 😅

Should be fixed now